docs/cn/Debug_In_Idea.md

```diff
@@ -26,9 +26,9 @@ deleteWhen = 04
 fileReservedTime = 48
 brokerRole = ASYNC_MASTER
 flushDiskType = ASYNC_FLUSH
-namesrvAddr = 127.0.0.1:9876 # name server地址
```

Comment:
Users copying this code block and running it will fail to start due to coding errors.
suggest deleting it.
`The Name Server Address[127.0.0.1:9876  # name serverå°å] illegal, please set it as follows, "127.0.0.1:9876;192.168.0.1:9876"`

Reply:
maybe you should not paste the commented content in the terminal